GB Locale files

hillzzz opened the pull request:
I have written the GB Locale files and inserted them into what I hope are the correct locations - these turn "Tax" into "VAT" "Cart" into "Basket" and enable UK English specific spellings etc. I hope it will help UK specific sites to get started more quickly. Thanks. Hillary

hillzzz committed:
GB Locale files

hillzzz committed:
GB Local Files

hillzzz committed:
GB Locale files

hillzzz committed:
Merge remote-tracking branch 'upstream/master' into develop

joshdifabio commented:
Good job! It'd probably be helpful if you committed the US English files first and then committed again with your changes; that way it'd be easy to see what you've changed :).

hillzzz commented:
Ah Ok - I will know for next time.

I have not made any changes to the en_US  files - all the changes I have made are in directories called “en_GB” so it should be clear as these directories don’t exist in the original Magento2 repository.

in other words if its in a directory called en_GB its mine and there are approximately 78 files.

Cheers

Hillary
07919 488425
zen71001@zen.co.uk



> On 3 Jul 2015, at 17:06, Josh Di Fabio <notifications@github.com> wrote:
> 
> Good job! It'd probably be helpful if you committed the US English files first and then committed again with your changes; that way it'd be easy to see what you've changed :).
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/magento/magento2/pull/1438#issuecomment-118383231>.
>

hillzzz committed:
Added missing en_GB.csv file to app/code/Magento/Checkouti18n/

hillzzz commented:
So its additional files in  the i18n/magento <https://github.com/magento/magento2/tree/develop/app/i18n/magento> directories, that I have added - path is for example, /app/i18n/magento <https://github.com/magento/magento2/tree/develop/app/i18n/magento>/en_gb/
I have just submitted another pull request because I missed an en_GB.csv file in app/code/Magento/Checkout/i18n/ <https://github.com/hillzzz/magento2/commit/f1cd1df930dee16baab28d53d23ddf6a7f79195f>en_GB.csv which is now added.


Cheers,

Hillary
07919 488425
zen71001@zen.co.uk



> On 3 Jul 2015, at 17:06, Josh Di Fabio <notifications@github.com> wrote:
> 
> Good job! It'd probably be helpful if you committed the US English files first and then committed again with your changes; that way it'd be easy to see what you've changed :).
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/magento/magento2/pull/1438#issuecomment-118383231>.
>

magento-cicd commented:
We have automated a Magento Contributor License Agreement verifier for contributions sent to our GitHub projects.
Please see the CLA agreement in the Pull Request comments.

Vinai commented:
The new required `registration.php` file that was introduced a few weeks back is missing, registering the locale with the component registrar. Otherwise the one failing integration test might not be directly caused by the new theme, but the reason for the failure needs to be researched.
Can you please update the PR accordingly? Thanks you!

benmarks commented:
@hillzzz Please check the CLA requirement & @Vinai's comment.

RohitSavaliya commented on app/code/Magento/AdminNotification/i18n/en_GB.csv:
```diff
@@ -0,0 +1,51 @@
+"Your web server is configured incorrectly. As a result, configuration files with sensitive information are accessible from the outside. Please contact your hosting provider.","Your web server is configured incorrectly. As a result, configuration files with sensitive information are accessible from the outside. Please contact your hosting provider."
+"We appreciate our merchants' feedback. Please <a href=""#"" data-mage-init=%1>take our survey</a> and tell us about features you'd like to see in Magento.","We appreciate our merchants' feedback. Please <a href=""#"" data-mage-init=%1>take our survey</a> and tell us about features you'd like to see in Magento."
+"Close popup","Close popup"
+Close,Close
```
hi you there?

KrystynaKabannyk commented:
Hi @hillzzz, are you still interested in your pull request? If yes, could you please sign License Agreement and update information?

davidalger commented:
Closing due to lack of reply from OP.
